src/quo/components/list_items/address/view.cljs

```diff
@@ -25,15 +25,16 @@
             [text/text
              {:size   :paragraph-1
               :weight :semi-bold
-              :style  {:color (colors/resolve-color network)}} (str (subs (name network) 0 3) ":")])
+              :style  {:color (colors/resolve-color network theme)}}
+             (str (subs (name network) 0 3) ":")])
```

Comment:
this data should come through as `short-name` from the `network-details sub 👍 
i.e `networks` has the shape 
```
[{:network-name :ethereum :short-name "eth"} {:network-name :optimism :short-name "opt"}]
```
can we use this approach here too? 🙏

Reply:
Done